### PR TITLE
Tab bar tab item mouse hover issue

### PR DESCRIPTION
**Pull request by Anubhvv**
### Problem 
1. Hover effect was not showing for Tab Items
2. DemoApp TabBar Activity Basic Text Text was not visible
### Fix
Defining the Correct Color values based on Focused State (Which is the color picked for Hover State).
Also Other components using TabItems will now see hover effect working fine.

### Validations
Manually tested and hover effect showing up, also attached the Screenshots
(how the change was tested, including both manual and automated tests)

### Screenshots

| Before                                       | After                                      |
|----------------------------------------------|--------------------------------------------|

| image    |  image  |

| image  |  image  |

| Screenshot or description before this change | Screenshot or description with this change |

### Pull request checklist

This PR has considered:
- [ ] Light and Dark appearances
- [ ] Automated Tests
- [ ] Documentation and demo app examples
- [ ] VoiceOver and Keyboard Accessibility
- [ ] Internationalization and RTL layouts
- [ ] Size classes and window sizes (notched devices, multitasking, different window sizes, etc)

### CODE REVIEW

File: fluentui_listitem/src/main/java/com/microsoft/fluentui/tokenized/tabItem/TabItem.kt
Comment (NamanPandey) [inline fluentui_listitem/src/main/java/com/microsoft/fluentui/tokenized/tabItem/TabItem.kt:74]: Remove extra space

File: fluentui_core/src/main/java/com/microsoft/fluentui/theme/token/controlTokens/TabItemTokens.kt
Comment (NamanPandey) [inline fluentui_core/src/main/java/com/microsoft/fluentui/theme/token/controlTokens/TabItemTokens.kt:56]: function name should start from small case

**Comment by Anubhvv** [comment]
Waiting for designer approval of colors

**Review by mishramayank1 — APPROVED** [review]

**Pull request merged**
